src/payment/checkout/payment-form/CardHolderInformation.jsx

```diff
@@ -37,7 +37,180 @@ export class CardHolderInformationComponent extends React.Component {
   }
 
   render() {
-    const { disabled, showBulkEnrollmentFields } = this.props;
+    const { disabled, showBulkEnrollmentFields, isPaymentVisualExperiment } = this.props;
+
+    if (isPaymentVisualExperiment) {
```

Comment:
Originally thought I was going to be making more changes here that we decided not to do, so now it looks a bit overkill. This section is long, but basically just a temporary copy of the old section with different translations, required status, and some different ordering/formatting